### PR TITLE
Email sender

### PR DESCRIPTION
**Pull request by adityamittl**
# Description

Fix Issue #31 

## Type of change

- [x] New Script (non-breaking change which adds a new script to the repository)

## Has This Been Tested?

- [x] I confirm that I have tested my script locally and verified that it works as intended.

## Checklist:

- [x] I have gone through the contributing guidelines before making this Pull Request.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: scripts/Email-Sender/script.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Script to automate Email Sending
+import smtplib
+
+to = input("Enter the Email of recipent:\n")
+content = input("Enter the Content for E-Mail:\n")
+
+sender_email = "youremail@mail.com"
+sender_password = "yourpassword"
```
Comment (OBITORASU) [inline scripts/Email-Sender/script.py:8]: Could you please add an input field for the user to provide this during runtime?

**Review by OBITORASU — CHANGES_REQUESTED** [review]

File: scripts/Email-Sender/script.py (hunk 1 above)
Comment (adityamittl) [reply scripts/Email-Sender/script.py:8]: Sure

**Comment by adityamittl** [comment]
Made the changes

**Comment by OBITORASU** [comment]
LGTM!

**Pull request merged**